### DISCUSSION TITLE
Horizontal scroll issue

### DISCUSSION THREAD

**Discussion by mass2527** [nick-keller/react-datasheet-grid#180] [Q&A] [answered]
Hello Mr. Keller,

My team is one of your library users and has been using your library for a long time. 
We appreciate your work due to it helps us a lot.

However, we have faced some issues of using it:

1. A horizontal scroll: it is bothering us checking the contents on the sheet since it hovers the contents over when there are numbers of columns.

Therefore, is there any ways that we could move the horizontal scroll under the datasheet?

<img width="993" alt="스크린샷 2022-09-02 오전 11 00 15" src="https://user-images.githubusercontent.com/70563791/188042760-6eaf56ef-d26a-4052-bba9-6d40b5bcc9fd.png">

**Comment by nick-keller** [comment, marked as answer]
Hey @mass2527 and thx for your comment!
I do believe you can still scroll vertically, the scrollbar should not "hide" content. The issue you are having is that the height you specify is the outter height, not the inner height.
To  remove the vertical scrollbar you'll need to add the height of the scrollbar to the [height](https://react-datasheet-grid.netlify.app/docs/api-reference/props#height) prop of the grid. To get the height of the scrollbar you can use a lib like [scrollbar-width](https://www.npmjs.com/package/scrollbar-width).
